project-base/tests/ShopBundle/Test/ParameterTransactionFunctionalTestCase.php

```diff
@@ -13,7 +13,7 @@ class ParameterTransactionFunctionalTestCase extends TransactionFunctionalTestCa
      * @param string $parameterValueNameId
      * @return int
      */
-    protected function getParameterValueId(string $parameterValueNameId): int
+    protected function getParameterValueIdForFirstDomain(string $parameterValueNameId): int
```

Comment:
Why we have it still hardcoded to specific domain?

What about passing a domainId as a second argument?

```suggestion
    protected function getParameterValueIdForDomain(string $parameterValueNameId, int $domainId): int
```
or maybe even better
```suggestion
    protected function getParameterValueIdForDomain(string $parameterValueNameId, \Shopsys\FrameworkBundle\Component\Domain\Domain $domain): int
```

Reply:
we do not need this